native/ios/Integreat.xcodeproj/xcshareddata/xcschemes/integreat-test-cms.xcscheme

```diff
@@ -27,6 +27,15 @@
             <ActionContent
                title = "Setup native locales"
                scriptText = "exec &gt; ${PROJECT_DIR}/prebuild_native_locales.log 2&gt;&amp;1&#10;cd ${PROJECT_DIR} &amp;&amp; ../tools/native-locales setup IntegreatTestCms --locales &quot;../../locales/locales.json&quot; --destination &quot;Integreat&quot;&#10;">
+               <EnvironmentBuildable>
+                  <BuildableReference
+                     BuildableIdentifier = "primary"
+                     BlueprintIdentifier = "13B07F861A680F5B00A75B9A"
+                     BuildableName = "IntegreatTestCms.app"
```

Comment:
Is this change on purpose?

Reply:
Yes, otherwise the locales preaction script is not run.

Reply:
Was still missing for IntegreatTestCms here, all other configs are already using it.